src/dos/cdrom_image.cpp

```diff
@@ -38,12 +43,32 @@
 #include <string.h>
 #endif
 
+#if defined(WORDS_BIGENDIAN)
+#define IS_BIGENDIAN true
+#else
+#define IS_BIGENDIAN false
+#endif
+
```

Comment:
I think define like this would be more appropriate in same location where `WORDS_BIGENDIAN` is defined. Or maybe it can be dropped altogether? Anyways: values `true` and `false` should be surrounded with the parentheses, as this is macro.

Reply:
Excellent; will make this change when we get to that "layer" in the sequence of merges.

Reply:
Figured out a way around using macros; it's pretty clean. You'll see it when we get to the layered PRs.  (I'm still working on this branch as my main sandbox to keep testing everything)